### PR TITLE
Change board serverRoot from '/plugins/focalboard' to '/boards'

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Changes the base URL for the server from '/plugins/focalboard' to '/boards'
This fixes the "Page not found" error in links when linking/unlinking a board. But does NOT fix the issue with Shared Boards.

#### Ticket Link
  Fixes https://github.com/mattermost/focalboard/issues/4441

### CODE REVIEW

**Review by wiggin77 — APPROVED** [review]

**Comment by ogi-m** [comment]
@sbishel I don't think I can test this in product mode before merge, so let's do that first.

**Comment by sbishel** [comment]
@mgdelacroix - Do you see any issues with this change? I don't think this overlaps with the non-authenticated urls issue, but want to make sure prior to merging.

File: mattermost-plugin/server/boards/boardsapp_util.go [hunk 1]
```diff
@@ -80,8 +80,12 @@ func createBoardsConfig(mmconfig mm_model.Config, baseURL string, serverID strin
 		showFullName = *mmconfig.PrivacySettings.ShowFullName
 	}
 
+	serverRoot := baseURL + "plugins/focalboard"
```
Comment (mgdelacroix) [inline mattermost-plugin/server/boards/boardsapp_util.go:83]:
```suggestion
	serverRoot := baseURL + "/plugins/focalboard"
```

**Review by mgdelacroix — CHANGES_REQUESTED** [review]
I don't think it should affect, as we're not serving the alternative webapp in product mode. Blocking on the slash comment as it breaks the plugin mode

File: mattermost-plugin/server/boards/boardsapp_util.go (hunk 1 above)
Comment (sbishel) [reply mattermost-plugin/server/boards/boardsapp_util.go:83]: I knew I wanted you to look for a reason...nice catch.

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

**Pull request merged**